src/server.ts

```diff
@@ -201,92 +235,128 @@ app.post("/callAnswered", async (req, res) => {
-    callId: bridgecallId,
-    // bridgeCompleteUrl: `${voiceCallbackUrl}/callStatus`,
-    bridgeTargetCompleteUrl: `${voiceCallbackUrl}/bridgeTargetComplete`,
+    sentence: "The call will start now",
   });
-
-  var response = new Response();
   response.add(speakSentence);
-  response.add(bridge);
 
-  let myResp = await response.toBxml();
-  console.log(myResp);
+  const bridgecallId: string = findCall("bridge");
+  if (bridgecallId) {
+    // the sipinterconnect call exists - bridge them
+    var bridge = new Bridge({
+      callId: bridgecallId,
+      bridgeTargetCompleteUrl: `${voiceCallbackUrl}/endBridgeLeg`,
+    });
+
+    response.add(bridge);
+  } else {
+    const pause = new Pause({
+      duration: 120,
+    });
+    response.add(pause);
+  }
+
+  let myResp: string = await response.toBxml();
+  console.log(`Bridging outbound call using Programmable Voice - ${callId}`);
+  console.log("BXML for the answered call: ", myResp);
 
   // Send the payload back to the Voice API
   res.send(myResp);
-  console.log(`Bridging outbound call using Programmable Voice - ${callId}`);
 });
 
-app.post("/callStatus", async (req, res) => {
-  res.status(200).send();
+/**
+ * the /bridgeCallAnswered api call completes the linkage of the webRTC and
+ * V2 Voice environments
+ */
+app.post("/bridgeCallAnswered", async (req, res) => {
+  const bridgeCallId = req.body.callId;
+  console.log(
+    `received answered callback SIP to WebRTC Bridge ${bridgeCallId} to ${req.body.to}`
+  );
 
-  console.log("Call Status update: ", req.body);
+  // preserve the call-leg
+  let data: CallData = {
+    from: req.body.from,
+    to: req.body.to,
+    callType: "bridge",
+  };
 
-  try {
-    if (req.body.eventType === "disconnect") {
-      // Do some cleanup
-      const callId = req.body.callId;
-      console.log(`received disconnect event for call ${callId}`);
+  voiceCalls.set(bridgeCallId, data); // preserve the info on the bridge leg in the calls map.
 
-      const callData = voiceCalls.get(callId);
-      if (callData?.bridge) {
-        // results from disconnecting the bridge - clean up
-        deleteParticipant(bridgeParticipant);
-      }
-      voiceCalls.delete(callId);
-    } else {
-      console.log("received unexpected status update", req.body);
-    }
-  } catch (e) {
-    console.log(`failed to cleanup departing participants...${e}`);
+  var response = new Response();
+  var speakSentence = new SpeakSentence({
+    sentence: `a call is happening`,
+  });
+
+  // if there is another call present, around bridge it in, otherwise wait for another call.
+  let otherCallId = findCall("outbound") || findCall("inbound"); // look for any calls that are hanging around.
+  if (otherCallId) {
+    // there is an existing voice call that should be bridged
+    console.log(
+      "bridge the preexisting voice call: ",
+      otherCallId,
+      voiceCalls.get(otherCallId)
+    );
+    // the sipinterconnect call exists - bridge them
```

Comment:
An external user isn't going to know what sipinterconnect is

Reply:
thanks